planetscale/images/IMAGES_README.md

```diff
@@ -0,0 +1,46 @@
+# Marketplace Media Carousel Guidelines
```

Comment:
Is this file supposed to be here? It looks it might be part of the integrations template that should be removed?

Reply:
Looks extraneous to me... removed.